assets/app/util/branchFormatter.js

```diff
@@ -0,0 +1,16 @@
+import {encodeB64, decodeB64 } from './encoding';
+
+// format draft branch anme
```

Comment:
i believe the function name and don't need this comment. I love how this code in this file has been extracted for reuse by others.